src/Components/Search/NewSearch/NewSearchBarFooter.tsx

```diff
@@ -1,35 +1,40 @@
 import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
 import { Flex, Text } from "@artsy/palette"
-import { themeGet } from "@styled-system/theme-get"
-import { RouterLink } from "System/Router/RouterLink"
 import { FC } from "react"
-import styled from "styled-components"
+import { useTracking } from "react-tracking"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import { SuggestionItemLink } from "./SuggestionItemLink"
+import { Highlight } from "./Highlight"
 
 interface SuggestionItemProps {
-  display: string
   href: string
-  imageUrl?: string
-  isHighlighted: boolean
-  label: string
   query: string
-  showArtworksButton?: boolean
-  showAuctionResultsButton?: boolean
+  onRedirect: () => void
 }
 
 export const NewSearchBarFooter: FC<SuggestionItemProps> = ({
   href,
-  isHighlighted,
   query,
+  onRedirect,
 }) => {
+  const tracking = useTracking()
+
   const handleClick = (
-    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+    _event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
   ) => {
-    event.preventDefault()
+    tracking.trackEvent({
+      action_type: DeprecatedSchema.ActionType.SelectedItemFromSearch,
```

Comment:
For new components, can we avoid using items from the deprecated analytics schema?

Reply:
Thanks! I've created a ticket to re-visit analytic events. For context, this component will be hidden behind a feature flag and won't be enabled after merging this PR. We have a couple of other tickets to implement before we can enable this feature (analytic events included)